documentation/development-docs/examples/get-cmdlet-example.md

```diff
@@ -0,0 +1,159 @@
+## `Get-*` cmdlet example
+
+_Note_: for the below examples, the string "TopLevelResource" would be replaced with the name of your top-level resource (_e.g._, "VirtualMachine", "VirtualNetwork", "SqlServer"), and the string "ChildResource" would be replaced with the name of your child resource (_e.g._, "VirtualMachineExtension", "VirtualNetworkPeering", "SqlDatabase")
+
+### Top-level resource
+
+All top-level resources should have a `Get-*` cmdlet that allows users to list the resources in their subscription or a resource group, as well as get a specific resource. In addition, users should be able to provide the resource id of the resource they want to get, and the cmdlet will parse the string to get the necessary identity information.
+
+#### Parameter sets
+
+To enable the scenarios mentioned previously, the cmdlet will need three parameter sets:
+
+```
+Get-AzTopLevelResource [-ResourceGroupName <String>]
```

Comment:
Ooh do we want to mention wildcard right now or just leave that to the generator?

Reply:
@maddieclayton let's leave wildcards for the generator